doc/CONTRIBUTING.md

```diff
@@ -1 +1 @@
-CONTRIBUTING.md
\ No newline at end of file
+/home/vkrm/cve-bin-tool/CONTRIBUTING.md
```

Comment:
You will have to create a symbolic link and not specify the path here.
Also, use the relative path. If you are on linux, this will do the job:
```
ln -s ../CONTRIBUTING.md CONTRIBUTING.md
```